Analytics/Integrations/Kahuna/SEGKahunaIntegration.h

```diff
@@ -22,6 +22,7 @@
 @property (nonatomic) NSDictionary *pushInfo;
 @property (nonatomic) UIApplicationState applicationState;
 @property (nonatomic) BOOL kahunaInitialized;
```

Comment:
The property is being accessed in Start (which is in background thread) as well as App Delegate notifications (which is in Main thread). So made the properties atomic.